catalog/datasets/aat.jsonld

```diff
@@ -25,10 +25,23 @@
       "@type": "DataDownload",
       "contentUrl": "http://vocab.getty.edu/sparql",
       "encodingFormat": "application/sparql-query",
-      "potentialAction": {
-        "@type": "SearchAction",
-        "query": "file://catalog/queries/aat.rq"
-      }
+      "potentialAction": [
+        {
+          "@type": "SearchAction",
+          "query": "file://catalog/queries/search/aat.rq"
+        },
+        {
+          "@type": "FindAction",
+          "query": "file://catalog/queries/lookup/aat.rq",
```

Comment:
To discuss: is this the appropriate @type? There is no `LookupAction` in Schema.org. `FindAction` could suffice, but it doesn't support the `query` property...

Reply:
At Schema.org [it says](https://schema.org/SearchAction):

> SearchAction generally leads to a FindAction.

Which is not quite the same as a lookup action, but that doesn’t exist in Schema. I’d say let’s just use FindAction for now for easy differentiation from the SearchAction, even if the former doesn’t allow the `schema:query` predicate.

Reply:
👍🏻